neuralmonkey/tf_manager.py

```diff
@@ -173,14 +174,21 @@ def execute(self,
                 train=False,
                 compute_losses=True,
                 summaries=True,
-                batch_size=None) -> List[ExecutionResult]:
+                batch_size=None,
+                log_progress: int = 0) -> List[ExecutionResult]:
         if batch_size is None:
             batch_size = len(dataset)
         batched_dataset = dataset.batch_dataset(batch_size)
+        log_time = time.process_time()
 
         batch_results = [
             [] for _ in execution_scripts]  # type: List[List[ExecutionResult]]
-        for batch in batched_dataset:
+        for batch_id, batch in enumerate(batched_dataset):
+            if (log_time + log_progress * 60 < time.process_time()
+                    and log_progress > 0):
+                log("Evaluated {}/{} examples.".format(batch_id * batch_size,
```

Comment:
To neni evaluated, ale processed. A nerozumim, proč je tam počet examplů lomeno aktuální čas procesu. Dal bych normálně jen počet examplů.